Utilities/bootstrap

```diff
@@ -42,7 +42,7 @@ logging.basicConfig(
     ]),
     level=logging.INFO,
 )
-g_macos_deployment_target = '12.0'
+g_macos_deployment_target = '14.0'
```

Comment:
Any chance we could tie this to DARWIN_DEPLOYMENT_VERSION_OSX in swiftlang/swift?

Reply:
I'm not sure how to do that tbh; I think that the best way to do that would be to get rid of this script and drive this process through the tooling in swiftlang/swift.